pytorch_toolkit/object_detection/model_templates/face-detection/face-detection-0200/modules.yaml

```diff
@@ -3,3 +3,4 @@ arg_converter: MMDetectionWiderArgsConverter
 trainer: MMDetectionTrainer
 evaluator: MMFaceDetectionEvaluator
 exporter: MMDetectionExporter
+config_transformers: [NNCFConfigTransformer]
```

Comment:
Why list?

Reply:
Because potentially we can use several config transformers to stack them.    
And the code for the list in this case is almost the same as the code for one transformer.

Although at the moment we have only one config transformer  implemented -- `NNCFConfigTransformer`.    
But I can think on some use cases in testing when it could be helpful.    
But we don't use the use cases yet.

So, if you want, I could remove the list here. The code will be almost the same.

Reply:
In which potential situation will we use more then one config transformer? If it is not hard for you, please add support of non-list case, when the only one transformer is present.

Reply:
The support of non-list case is added.

Reply:
The code was significantly simplified, so now no lists here.